zetaclient/zeta_supply_checker.go

```diff
@@ -150,27 +149,47 @@ func (zs *ZetaSupplyChecker) CheckZetaTokenSupply() error {
 	return nil
 }
 
+type ZetaSupplyCheckLogs struct {
+	Logger                   zerolog.Logger
+	AbortedTxAmounts         sdkmath.Int `json:"aborted_tx_amounts"`
+	ZetaInTransit            sdkmath.Int `json:"zeta_in_transit"`
+	ExternalChainTotalSupply sdkmath.Int `json:"external_chain_total_supply"`
+	ZetaTokenSupplyOnNode    sdkmath.Int `json:"zeta_token_supply_on_node"`
+	EthLockedAmount          sdkmath.Int `json:"eth_locked_amount"`
+	NodeAmounts              sdkmath.Int `json:"node_amounts"`
+	Lhs                      sdkmath.Int `json:"lhs"`
```

Comment:
What does it mean? We should use a more explicit name for the attribute

Reply:
The left-hand side and right-hand side are just for internal use.

Reply:
Even if it is for internal use, I think it is still better to make explicit name or provide comment explaining